src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -25,13 +31,24 @@ interface PharmacyProps {
 export function PharmacyCard(props: PharmacyProps) {
   const [booking, setBooking] = useState(props.booking);
   const [lastUpdated, setlastUpdated] = useState(props.lastUpdated);
+  const [shouldShowMap, setShouldShowMap] = useState(false);
 
   const updatePharmacy = (availability: boolean) => {
     const time = new Date();
     setlastUpdated(time.toLocaleString());
     setBooking(availability);
   };
 
+  const Map = () => (
+    <Iframe
+      // the key is hardcoded for now, but there's no rate limit or usage limit for embedding
+      url={`https://www.google.com/maps/embed/v1/place?q=${props.address}&key=AIzaSyCJF0WPrXAbLIePWWFbS7rRxdCBaY8pjAs`}
+      width="100%"
+      height="100%"
```

Comment:
The height shows pretty small, it's hard to actually see much on the map

Reply:
What do you suggest, style-wise?

Reply:
UI/UX isn't my thing so I'm prolly the wrong one to suggest values, but can we apply a different heights via media queries?

Reply:
I think how it is gives a good enough idea of where it is in someones city. They can also just click the map to see more. Making it any bigger would make it much harder to navigate on mobile (where most people will be using this), so I think just leaving the height how it is, and letting them just click the map if they want to see more should be fine